### ISSUE TITLE
More examples

### ISSUE THREAD

**Issue by objectiveSee** [jfromaniello/express-unless#23]
It would be nice to see more examples of usage in this README. I am stuck trying to figure a lot of use cases out. :)

**Comment by KrisLau** [comment]
I agree I would love to see some regex use cases especially for common ones like excluding all sub-routes under the same path. For example, excluding `/user` and all its sub-paths like `/user/:id`, `/user/groups`, etc
